demisto_sdk/commands/common/content/objects/pack_objects/readme/readme.py

```diff
@@ -27,6 +28,17 @@ def mention_contributors_in_readme(self):
         except Exception as e:
             print(e)
 
+    def handle_marketplace_tags(self):
+        try:
+            with open(self._path, 'r+') as f:
+                text = f.read()
+                f.seek(0)
+                f.write(MARKETPLACE_TAG_PARSER.parse_text(text))
+                f.truncate()
+        except Exception as e:
+            print(e)
```

Comment:
don't we want to raise it? or at least print_error?

Reply:
Was lazy and copied what the above method does. I think in this case a normal print should be ok, but it's better to log it properly.